app/controllers/courses/applications_open_controller.rb

```diff
@@ -4,13 +4,15 @@ class ApplicationsOpenController < ApplicationController
     before_action :build_course_params, only: %i[update continue]
     include CourseBasicDetailConcern
 
+    # rubocop:disable Lint/UselessMethodDefinition
```

Comment:
Do we need these methods? Not even sure what they're supposed to be doing. 

If so it's probably better to add these to a rubocop_todo file (or exclude this file in rubocop.yml if we're never going to fix it). Better to keep it in one place than comment throughout the code. Same for all the other ones. You can autogenerate the file if you remove the comments.

Reply:
Removing the methods creates another rubocop offense, but it's a good idea to add it to a rubocop_todo file. Ill do it now 👍